### PR TITLE
Update authorization metadata and SMART authorization links

### PR DESCRIPTION
**Pull request by daliboz**
Update that authorization metadata and some SMART links that were broken.

### CODE REVIEW

**Comment by icon5585** [comment]
+1

File: lib/resources.rb [hunk 1]
```diff
@@ -877,259 +877,390 @@ def get_resource(key)
...
-                      {
-                        "name"=> "patient",
-                        "type"=> "reference",
-                        "documentation"=> "The patient for the vaccination record"
+        "resourceType"=> "Conformance",
+        "text"=> {
+            "status"=> "generated",
+            "div"=> "\u003cdiv\u003eGenerated Conformance Statement\u003c/div\u003e"
+        },
+        "url"=> "https=>//fhir.sandboxcernerpowerchart.com/may2015/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Conformance",
```
Comment (dennispatterson) [inline lib/resources.rb]: There's a find/replace issue with the protocol on some of these urls (e.g. "https=>//fhir....").

File: lib/resources.rb [hunk 2]
```diff
@@ -877,259 +877,390 @@ def get_resource(key)
...
+        "name"=> "Cerner Conformance Statement",
+        "publisher"=> "Cerner",
+        "description"=> "Describes capabilities of this server",
+        "status"=> "draft",
+        "date"=> "2015-10-09T19=>12=>45.663+00=>00",
```
Comment (dennispatterson) [inline lib/resources.rb]: Similar to above, date has '=>' embedded.

File: lib/resources.rb [hunk 3]
```diff
@@ -877,259 +877,390 @@ def get_resource(key)
...
-                    ],
-                    "searchParam"=> [
-                      {
-                        "name"=> "patient",
-                        "type"=> "reference",
-                        "documentation"=> "The patient for the vaccination record"
+        "resourceType"=> "Conformance",
+        "text"=> {
+            "status"=> "generated",
+            "div"=> "\u003cdiv\u003eGenerated Conformance Statement\u003c/div\u003e"
```
Comment (dennispatterson) [inline lib/resources.rb]: This is not rendering the brackets around the div tag.

File: content/may2015.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ Vary: User-Agent
...
...erence","documentation":"The subject that the observation is about"},{"name":"code","type":"token","documentation":"The type of observation"},{"name":"date","type":"date","documentation":"The date range into which the observation falls. This parameter value should be present twice, prefixed once by '\u003e' representing the earliest date and once by '\u003c' representing the latest date."}]},{"type":"Patient","interaction":[{"code":"read"},{"code":"search-type"}],"readHistory":false,"searchParam":[{"name":"_id","type":"token","documentation":"The logical resource id associated with the resource (must be supported by all servers)"},{"name":"birthdate","type":"date","documentation":"The patient's date of birth"},{"name":"identifier","type":"token","documentation":"A patient identifier"},{"name":"name","type":"string","documentation":"A portion of either family or given name of the patient"},{"name":"telecom","type":"string","documentation":"The value in any kind of telecom details of the patient"},{"name":"start","type":"number","documentation":"The offset to use when returning results"},{"name":"_count","type":"number","documentation":"The maximum number of results to return"}]}]}]}
```
Comment (dennispatterson) [inline content/may2015.md]: Besides what's already been stated, in the documentation for Observation's date parameter, '\u003e' and '\u003c' are not rendered as brackets.

Comment (MaxPhilips) [inline content/may2015.md]: Would we also be interested in prettifying this or is it preferred to leave the response on one line to reduce vertical scrolling?

File: lib/resources.rb [hunk 5]
```diff
@@ -877,259 +877,390 @@ def get_resource(key)
...
+                                                             "type"=> "reference",
+                                                             "documentation"=> "The subject that the observation is about"
+                                                         }, {
+                                                             "name"=> "code",
+                                                             "type"=> "token",
+                                                             "documentation"=> "The type of observation"
+                                                         }, {
+                                                             "name"=> "date",
+                                                             "type"=> "date",
+                                                             "documentation"=> "The date range into which the observation falls. This parameter value should be present twice, prefixed once by '\u003e' representing the earliest date and once by '\u003c' representing the latest date."
```
Comment (MaxPhilips) [inline lib/resources.rb]: Dennis mentioned this for content/may2015.md but noting that \u003e and \u003c need to be changed to > and < here too

File: content/may2015.md (hunk 4 above)
Comment (MaxPhilips) [inline content/may2015.md]: There's a \u003c for MedPrescription date too.

**Comment by daliboz** [comment]
Checked in corrections.

File: content/may2015.md (hunk 4 above)
Comment (dennispatterson) [inline content/may2015.md]: Instances of '\u003e' and '\u003c' are not rendered as angled brackets

Comment (daliboz) [inline content/may2015.md]: @dennispatterson I'm not finding the unicode characters anymore (see updated commit below).

Comment (dennispatterson) [inline content/may2015.md]: Disregard.  I was referring more to the output of the example curl command to retrieve metadata, but this displays unicode characters when executed anyway.

**Comment by dennispatterson** [comment]
:+1:

**Comment by MaxPhilips** [comment]
+1

**Comment by jfrhong** [comment]
+1

**Comment by icon5585** [comment]
+1

**Pull request merged**